### DISCUSSION TITLE
Nabu Casa rewritten and reorganized their upstream Silabs firmware builder

### DISCUSSION THREAD

**Discussion by Hedda** [darkxst/silabs-firmware-builder#98] [General]
FYI, it looks like puddly has in the past few weeks rewritten and reorganized much of Nabu Casa’s upstream Silabs firmware builder, and even added Z-Wave Controller too, plus archived their repo for binaries:

https://github.com/NabuCasa/silabs-firmware-builder/pulls?q=is%3Apr+is%3Aclosed

https://github.com/NabuCasa/silabs-firmware-builder/commits/main/

> [!IMPORTANT]  
> **_New firmwares are released straight from the [SiLabs Firmware Builder repository](https://github.com/NabuCasa/silabs-firmware-builder/releases)._**
> **_This repository contains all past firmwares and will not receive future updates._**

https://github.com/NabuCasa/silabs-firmware

So, new official Zigbee and Thread firmware releases for SkyConnect + Yellow and future Silicon Labs firmware will from now be released straight from the **[SiLabs Firmware Builder repository](https://github.com/NabuCasa/silabs-firmware-builder/releases)**

As such the "old" **[silabs-firmware](https://github.com/NabuCasa/silabs-firmware)** repository is now listed as archived and will not receive more updates but it will remain available as it contains all past firmware versions.

PS: Off-topic however interesting is that the **[silabs-firmware-builder](https://github.com/NabuCasa/silabs-firmware-builder)** now also contain Z-Wave Controller firmware for the upcoming official Home Assistant Z-Wave Controller USB adapter hardware that Nabu Casa is working on -> https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead

**Comment by darkxst** [comment]
Yes, I have been following this, will sync up with this work on this builder once I get a chance.

**Comment by Hedda** [comment]
Might they need to make major changes now with the release of Gecko Platform Version 5.0.0.0 and the migration to new Simplicity SDK?

- https://github.com/darkxst/silabs-firmware-builder/discussions/100
